setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import os
+
+from pathlib import Path
+from typing import List
+
+from codecs import open
+from setuptools import find_packages, setup
+
+
+here = Path(__file__).parent
+
+
+def parse_requirements(filename: str) -> List[str]:
+    reqs = (here / filename).read_text().strip().split('\n')
+    reqs = [r.strip() for r in reqs]
+    reqs = [r for r in sorted(reqs) if r and not r.startswith('#')]
+    return reqs
+
+
+# Load the package's __init__.py file as a dictionary.
+pkg = {}
+with open(here / 'aionetbox' / '__init__.py', 'r', 'utf-8') as f:
+    exec(f.read(), pkg)
```

Comment:
I don't think this will work because of the import. this is what I have done to get around that: https://github.com/edaniszewski/sanic-healthcheck/blob/master/setup.py#L14

Reply:
ah - I see you addressed this with a different approach elsewhere https://github.com/vapor-ware/prophetess-netbox/commit/01b783107f1dd048e7aacc96532591921baf64d8

that could work here too. just note that if you define something like `__all__` in the init, this approach won't work, but since we're not doing that, I think this would work here as well.

Reply:
I like this better approach better than an exec